### PR TITLE
add docker files for dockerizing store project

### PR DESCRIPTION
**Pull request by azabolotnaiaGD**

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -1,2 +1,2 @@
-
+server.port=9090
```
Comment (acostasgd) [inline src/main/resources/application.properties]: Why requires to update the web application port.

Comment (azabolotnaiaGD) [reply src/main/resources/application.properties]: Because docker runs on 8080

Comment (acostasgd) [reply src/main/resources/application.properties]: Docker is not a service, the container itself doesn't occupy any port, only the service which is running on the container.

**Review by acostasgd — APPROVED** [review]

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      - springbootmysql-net
+    environment:
+      - MYSQL_ROOT_PASSWORD=password
+      - MYSQL_DATABASE=store_project_db
+      - MYSQL_PASSWORD=password
+
+  springboot-app:
+    image: springboot-store
+    ports:
+      - "9090:8080"
```
Comment (DrVa97) [inline docker-compose.yml:15]: Is this still necessary?

Comment (azabolotnaiaGD) [reply docker-compose.yml:15]: ports?

Comment (DrVa97) [reply docker-compose.yml:15]: Sorry for the confusing comment. Does it need to be 9090 or could it be 8080?

Comment (azabolotnaiaGD) [reply docker-compose.yml:15]: We need 9090 because we decided to run our application locally on 8080 and in docker on 9090

**Review by DrVa97 — APPROVED** [review]

**Review by anovosiolovGD — APPROVED** [review]

**Pull request merged**